deploy/docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM alpine:3.5
+
+LABEL maintainer="adundovi@gmail.com"
+LABEL description="Prepares a docker image \
```

Comment:
OK. I didn't pay attention to these lines and of course that everything which has some reasons behind is a better choice. Also, if you have an official pump.io e-mail address, we can put it there.